src/transformers/models/codegen/tokenization_codegen_fast.py

```diff
@@ -177,6 +176,35 @@ def _encode_plus(self, *args, **kwargs) -> BatchEncoding:
 
         return super()._encode_plus(*args, **kwargs)
 
+    def create_token_type_ids_from_sequences(
```

Comment:
let's use copied from here! (from the slow code)

Reply:
I have added `Copied from ...` comment!
(Please let me know if this change is not what you intended)

Reply:
yes! 🤗